test/TestDAOSetup.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.13;
+pragma solidity >=0.5.0;
```

Comment:
This version constraint is safe here AFAIK but we may forget to update it if we ever add arithmetic to these tests
I'd recommend having `>= 0.5.0` in interfaces and `^0.8.0` in tests (we know most tests won't fully work as expected before `0.8.0` because of arithmetic and we don't know if they'll fully work as intended in `0.9.0`)
The point from my original comment was that we know these tests work in versions `0.8.0` to `0.8.13` so I didn't understand the reason we're constraining `^0.8.13`